12-wt-social-network-analysis.Rmd

```diff
@@ -100,7 +99,6 @@ Let's put these into their own data frame, called `mentions`.
 ```{r}
 mentions <-
   tt_tweets %>%
-  filter(has_mention) %>%
```

Comment:
I got the same `mentions` dataframe before and after removing `has_mention`

Reply:
ahh good catch. took a look - this looks like it's a product of `unnest()`, which only retains rows that are not length 0 (the rows with `character(0)` - those without any mentions - are automatically removed). in short, good catch, thanks for removing this.